src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -289,6 +291,15 @@ public without sharing class UTIL_OrgTelemetry_SVC {
         featureManager.setPackageBooleanValue(TelemetryParameterName.Org_IsEnabled_AdvancedMultiCurrency.name(), isEnabled);
     }
 
+    /**
+     * @description Checks the Allocation Settings Custom Setting to determine if the Payment Allocation option is enabled. 
+     */
+    private void handleIsPaymentAllocationEnabled() {
+        Allocations_Settings__c allocationCustomSetting = UTIL_CustomSettingsFacade.getAllocationsSettings();
+        
+        featureManager.setPackageBooleanValue(TelemetryParameterName.IsEnabled_PaymentAllocation.name(), allocationCustomSetting.Payment_Allocations_Enabled__c);
```

Comment:
@KietTruongSFDO Since Boolean fields in custom settings can be null, I recommend using `== true` for this condition.